.github/workflows/lint_and_test_cpp.yaml

```diff
@@ -373,6 +373,90 @@ jobs:
           ./scripts/config_hdfs.sh
           ./run.sh test --on_travis -m ${{ matrix.test_module }}
 
+  build_with_jemalloc:
+    name: Build with jemalloc
+    needs: cpp_clang_format_linter
+    runs-on: ubuntu-latest
+    container:
+      image: apache/pegasus:thirdparties-bin-test-jemallc-ubuntu1804-${{ github.base_ref }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup cache
+        uses: actions/cache@v3
+        with:
+          path: |
+            /github/home/.ccache
+          key: release_ccache
```

Comment:
Since the cache with the same name will overwrite the other, so it's better to use another name for jemalloc build type cache, such as `jemalloc_ccache`

Reply:
> Since the cache with the same name will overwrite the other, so it's better to use another name for jemalloc build type cache, such as `jemalloc_ccache`

OK, I'll change the name.